### PR TITLE
feat: add header for E-commerce project

### PR DESCRIPTION
**Pull request by Vikas2426**
image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.5rem 2rem;
+  font-weight: 700;
+  background-color: #ffffff;
```
Comment (manismk) [inline style.css:20]: use variables here also

**Review by manismk — APPROVED** [review]

File: component-library/badge.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  align-items: center;
+  justify-content: center;
+  border: none;
+  margin: 0rem 1rem;
+}
+
+.badge {
+  position: absolute;
+  width: 20px;
+  height: 20px;
```
Comment (Pratik1005) [inline component-library/badge.css:17]: Just a suggestion: use rems as much as possible.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+input {
+  width: 30vw;
+  margin-left: 10px;
+  font-size: large;
+}
+input,
+input:focus {
+  border: none;
+  outline: none;
+}
```
Comment (Pratik1005) [inline style.css:53]: if possible don't directly target elements use class for styling

**Review by Pratik1005 — APPROVED** [review]
I have suggested some changes, you can update it if you get time.

File: component-library/badge.css (hunk 2 above)
Comment (Vikas2426) [reply component-library/badge.css]: done

File: style.css (hunk 3 above)
Comment (Vikas2426) [reply style.css:53]: need it for every input element that's why used element selector

File: style.css (hunk 1 above)
Comment (Vikas2426) [reply style.css]: fixed

**Pull request merged**